web/src/bootstrap_typeahead.ts

```diff
@@ -554,6 +579,11 @@ export class Typeahead<ItemType extends string | object> {
     }
 
     keyup(e: JQuery.KeyUpEvent): void {
+        // NOTE: Ideally we can ignore meta keyup calls here but
+        // it's better to just trigger the lookup call to update the list in case
+        // it did modify the query. For example, `Command + delete` on Mac
+        // doesn't trigger a keyup event but when `Command` is released, it
+        // triggers a keyup event which correctly updates the list.
```

Comment:
Keyup handlers are usually buggy and should be migrated to use more modern handlers, but that's it's whole own rabbit hole.

Reply:
Yeah, I tried changing it but it leads to whole new class of bugs.